filename: 0conftest.py, smelly_lines: 54 (Empty exception detected)
: 168format_stack.py, LINE: 49 (Too general exception)
FILENAME: 168format_stack.py, LINE: 132 (Empty exception detected)
FILENAME: 168format_stack.py, LINE: 339 (Empty exception detected)
FILENAME: 169memory.py, LINE: 162 (Empty exception detected)
FILENAME: 170func_inspect.py, LINE: 68 (Too general exception)
FILENAME: 175_dask.py, LINE: 81 (Empty exception detected)
FILENAME: 1setup.py, LINE: 23 (Empty exception detected)
FILENAME: 211spawn.py, LINE: 72 (Empty exception detected)
FILENAME: 230test_config.py, LINE: 54 (Empty exception detected)
FILENAME: 276multiclass.py, LINE: 265 (Empty exception detected)
FILENAME: 290testing.py, LINE: 64 (Empty exception detected)
FILENAME: 290testing.py, LINE: 76 (Empty exception detected)
FILENAME: 290testing.py, LINE: 768 (Empty exception detected)
FILENAME: 307test_estimator_checks.py, LINE: 324 (Empty exception detected)
FILENAME: 307test_estimator_checks.py, LINE: 367 (Too general exception)
FILENAME: 404test_split.py, LINE: 1159 (Empty exception detected)
FILENAME: 405test_validation.py, LINE: 525 (Empty exception detected)
FILENAME: 405test_validation.py, LINE: 971 (Empty exception detected)
FILENAME: 405test_validation.py, LINE: 1642 (Empty exception detected)
FILENAME: 406test_search.py, LINE: 748 (Empty exception detected)
FILENAME: 51test_classification.py, LINE: 1976 (Empty exception detected)
FILENAME: 699bench_rcv1_logreg_convergence.py, LINE: 42 (Too general exception)
FILENAME: 699bench_rcv1_logreg_convergence.py, LINE: 51 (Too general exception)
FILENAME: 699bench_rcv1_logreg_convergence.py, LINE: 56 (Too general exception)
FILENAME: 741parse_path.py, LINE: 57 (Too general exception)
FILENAME: 742pyparsing.py, LINE: 4422 (Empty exception detected)
